pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -801,10 +825,12 @@ PartitionedTopicInternalStats getPartitionedInternalStats(String topic)
      *            topic name
      * @param subName
      *            Subscription name
+     * @param force
+     *            Delete topic forcefully
      *
      * @return a future that can be used to track when the subscription is deleted
      */
-    CompletableFuture<Void> deleteSubscriptionAsync(String topic, String subName);
+    CompletableFuture<Void> deleteSubscriptionAsync(String topic, String subName, boolean force);
```

Comment:
This is an API breaking change. You can't directly change the method signature. Please add an override method.

Reply:
Added, thanks.